apps/squzy_storage/main.go

```diff
@@ -12,11 +13,26 @@ import (
 	"github.com/squzy/squzy/internal/logger"
 	apiPb "github.com/squzy/squzy_generated/generated/github.com/squzy/squzy_proto"
 	"google.golang.org/grpc"
+	"io/ioutil"
 )
 
 func main() {
+	var cfg config.Config
+
+	filename := flag.String("config", "", "path to configFile")
+	if filename == nil || *filename == "" {
```

Comment:
you should use this approach
`filename == nil || filename == nil && *filename == ""`

|| will mean it can check if `nil pointer == ""`

Reply:
also you can remove `else` and just have 
cfg = config.New() by default

and replace if with 

`if !(filename == nil || filename == nil && *filename == "") {
cfgFromFile, err := readConfigFile(filename)`

Reply:
Yea, I was thinking about that, and then I supposed, that it's better to make behaviour more explcict. As user I would prefer to know, if my config is read from os.env, for the case when I've forgotten to mention this flag.